frame/ethereum/src/lib.rs

```diff
@@ -265,15 +309,17 @@ impl<T: Trait> Module<T> {
 				}
 			},
 			ethereum::TransactionAction::Create => {
-				let contract_address = pallet_evm::Module::<T>::execute_create(
-					source,
-					transaction.input.clone(),
-					transaction.value,
-					transaction.gas_limit.low_u32(),
-					Some(transaction.gas_price),
-					Some(transaction.nonce),
-					true,
-				).unwrap().1; // TODO: handle error
+				let contract_address = Self::handle_exec(
+					pallet_evm::Module::<T>::execute_create(
+						source,
+						transaction.input.clone(),
+						transaction.value,
+						transaction.gas_limit.low_u32(),
+						Some(transaction.gas_price),
+						Some(transaction.nonce),
+						true,
+					).unwrap()
+				).unwrap().1;
```

Comment:
We need to eventually remove any `unwrap` in the code, but I guess that's out of scope of this PR.